app/views/articles/index.html.erb

```diff
@@ -1,11 +1,9 @@
 <h1>Hello, Rails!</h1>
 <ul>
   <% @articles.each do |article|%>
-    <% unless article.archived? %>
-     <li>
-       Title=> <%= link_to article.title, article %>  Body=><%= link_to article.body, article %><br><br>
-     </li>
-    <% end %>
- <% end %>
+   <li>
+      Title=>  <%= link_to article.title, article %><br> Body=>  <%= link_to article.body, article %><br><br>
```

Comment:
Can we move this `_article.html.erb` partial?

Reply:
why should we move this to  partial?